HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -358,6 +372,7 @@ instance_group [
 ]
 
 EOF
+	fi
 
 	if [[ $PLATFORM == *"onnx"* ]]; then
```

Comment:
should now be elif

Reply:
I think this if check is separate from the ensemble or !ensemble check.  If you prefer, I could put these in the above else !ensemble case

Reply:
yes, either put these in the !ensemble case, or put a return statement at the end of the ensemble case to indicate that the subsequent cases should never occur

Reply:
Ok I moved these inside of the !ensemble case